devel/uv/Portfile

```diff
@@ -4,11 +4,11 @@ PortSystem              1.0
 PortGroup               cargo 1.0
 PortGroup               github 1.0
 
-github.setup            astral-sh uv 0.1.13
+github.setup            astral-sh uv 0.1.15
 github.tarball_from     archive
 revision                0
 categories              devel python
-license                 Apache-2 MIT
+license                 {Apache-2 MIT}
```

Comment:
the change implies that one can pick either of the two licenses. Is that indeed the intention or do both apply?